vyper/exceptions.py

```diff
@@ -134,6 +135,9 @@ def format_annotation(self, value):
             if fn_node:
                 node_msg = f'{node_msg}function "{fn_node.name}", '
 
+        if self.path is not None:
+            node_msg = f'{node_msg}contract "{self.path}", '
```

Comment:
code smell that this code is similar with line 132 above -- and in fact, line 132 appends lineno but this does not. also the path could be appended twice in case there is both a node and `.path` set on this exception

Reply:
is it better now?